jwt/src/main/java/me/uport/sdk/jwt/JWTTools.kt

```diff
@@ -295,27 +312,50 @@ class JWTTools(
             PublicKey(pkBytes.toBigInteger()).normalize()
 
         }.filter { publicKey ->
+            try {
+                ecVerify(messageHash, sigData, publicKey)
```

Comment:
If the publicKey can't be decoded this method would throw, short-circuiting the other keys in the list and falsely stating that the JWT is invalid.
I added a try/catch to handle this error.

Reply:
Nice catch